app/scripts/migrations/120.2.ts

```diff
@@ -40,7 +41,7 @@ function removeObsoleteSnapControllerState(
   if (!hasProperty(state, 'SnapController')) {
     return;
   } else if (!isObject(state.SnapController)) {
-    global.sentry.captureException(
+    global.sentry?.captureException?.(
```

Comment:
Updating these to follow the convention in other migrations of not assuming this global is set. It should always be set in production, but one of the migrator tests doesn't set it and can blow up here.

Discovered while investigating this test failure:
https://github.com/MetaMask/metamask-extension/actions/runs/10216054639/job/28266703200